ci/before_install.sh

```diff
@@ -1,36 +1,60 @@
-set -ex
+set -e
```

Comment:
I think having `-x` is useful for debugging CI runs -- is there a particular reason you removed it?

Reply:
Not really, the script was taken from clang-sys, and they removed it upstream. But I'll re-add.